### ISSUE TITLE
Add a Button to bring out the authentication credentials dialog

### ISSUE THREAD

**Issue by Kemyikpe** [opendatakit/tool-suite-X#436]
**Description:**
Upon filling the server setting there should be a button for user to click to bring out the authentication credentials dialog.

**Proposed fix:**
Add a button *Proceed* that will bring out the Authentication Credentials Dialog when clicked.

The text on the button could be *Proceed* (I'm open to button name suggestions here)

![button](https://github.com/odk-x/tool-suite-X/assets/107570182/23d915fa-2fe8-4219-9c41-70ef1589f853)

**Why it's needed:**
It will give a better user experience, making users not confused on how to proceed from the screen after filling the server settings form.

**Comment by Lamouresparus** [comment]
@wbrunette what do you think?

**Comment by wbrunette** [comment]
@Lamouresparus This is someone filing an issue for possible UI design which is part of the process for UI design.

**Comment by Redeem-Grimm-Satoshi** [comment]
> **Description:** Upon filling the server setting there should be a button for user to click to bring out the authentication credentials dialog.
> 
> **Proposed fix:** Add a button _Proceed_ that will bring out the Authentication Credentials Dialog when clicked.
> 
> The text on the button could be _Proceed_ (I'm open to button name suggestions here)
> 
> ![button](https://user-images.githubusercontent.com/107570182/273820415-23d915fa-2fe8-4219-9c41-70ef1589f853.jpeg)
> 
> **Why it's needed:** It will give a better user experience, making users not confused on how to proceed from the screen after filling the server settings form.

@Kemyikpe Is this going to replace the back navigation?

**Comment by cvemann** [comment]
Hi @Kemyikpe, there's an ongoing issue that seeks to address what you've raised here. See #425

**Comment by Kemyikpe** [comment]
> > **Description:** Upon filling the server setting there should be a button for user to click to bring out the authentication credentials dialog.
> > **Proposed fix:** Add a button _Proceed_ that will bring out the Authentication Credentials Dialog when clicked.
> > The text on the button could be _Proceed_ (I'm open to button name suggestions here)
> > ![button](https://user-images.githubusercontent.com/107570182/273820415-23d915fa-2fe8-4219-9c41-70ef1589f853.jpeg)
> > **Why it's needed:** It will give a better user experience, making users not confused on how to proceed from the screen after filling the server settings form.
> 
> @Kemyikpe Is this going to replace the back navigation?

Yes, that is the idea @Redeem-Grimm-Satoshi, so that users don't have to be thinking and searching for how to move to next screen. I'll be using the [ODX-X Design button guides](https://docs.google.com/presentation/d/1Ol7YhG4WPLOWCoQZUrKHLPguHSiDz8RC3m8dNyTGGLo/edit#slide=id.g1dc0e432633_0_0) for the button.

**Comment by Kemyikpe** [comment]
> Hi @Kemyikpe, there's an ongoing issue that seeks to address what you've raised here. See #425

Hi @Cveman1 I see that the issue has not been approved yet
